packages/voiceflow-types/src/node/index.ts

```diff
@@ -3,6 +3,7 @@ import { BaseNode } from '@voiceflow/base-types';
 import * as Buttons from './buttons';
 import * as Capture from './capture';
 import * as CaptureV2 from './captureV2';
+import * as CardV2 from './cardV2';
```

Comment:
this seems duplicated with line 15

Reply:
Actually, line 15 is an export. They're all following this pattern (: